### ISSUE TITLE
Python: use numpy to wrap C++ arrays 

### ISSUE THREAD

**Issue by dbrnz** [opencor/libopencor#3]
This should be self obvious. However for some reason Zinc's Python wrappers expose numerical arrays as Python lists, which IMHO is not the best way to do things, so raising this issue...

**Comment by agarny** [comment]
Yes, we definitely want to make our Python bindings feel "natural" and in this case I agree that this would indeed mean using numpy rather than a Python list.

**Comment by agarny** [comment]
This has been done. libOpenCOR now has a depedency on `numpy` since simulation results are now retrieved as NumPy arrays (see PR [#576](https://github.com/opencor/libopencor/pull/576)).
